packages/Toast/src/Toast.js

```diff
@@ -0,0 +1,143 @@
+import React from "react";
+import PropTypes from "prop-types";
+
+import CheckIcon from "@paprika/icon/lib/Check";
+import CautionIcon from "@paprika/icon/lib/Caution";
+import ExclamationCircleIcon from "@paprika/icon/lib/ExclamationCircle";
+import InfoCircleIcon from "@paprika/icon/lib/InfoCircle";
+import LockIcon from "@paprika/icon/lib/Lock";
+
+import toastStyles, { contentStyles, IconStyled, CloseButtonStyled } from "./Toast.styles";
+
+const propTypes = {
+  /**
+   * Will make this an "assertive" alert that will interrupt immediately.
+   * Default behaviour is to be "polite" and wait until the user is idle.
+   */
+  ariaAlert: PropTypes.bool,
+
+  /** Will automatically close after 1500ms (or longer if provided by autocloseTimeout) */
```

Comment:
The actual default value is 5000, not 1500.  Not sure if that should be updated at this time? Do any designers have an opinion?

Reply:
Oh my bad, I was testing, it should be 1500ms